versions/3.1.0.md

```diff
@@ -2276,11 +2276,16 @@ This object is an extended subset of the [JSON Schema Specification Wright Draft
 For more information about the properties, see [JSON Schema Core](https://tools.ietf.org/html/draft-wright-json-schema-00) and [JSON Schema Validation](https://tools.ietf.org/html/draft-wright-json-schema-validation-00).
 Unless stated otherwise, the property definitions follow the JSON Schema.
 
-##### Properties 
+##### Properties
 
-The following properties are taken directly from the JSON Schema definition and follow the same specifications:
+OpenAPI v3.1 is a JSON Schema vocabulary which extends JSON Schema Core and Validation vocabularies. As such any keyword available for those vocabularies is by definition available in OpenAPI, and will work the exact same way, including but not limited to:
```

Comment:
I don't think it is correct to say that "OpenAPI v3.1 is a JSON Schema vocabulary".  We could say that the OpenAPI Schema object is implemented using a JSON Schema vocabulary.

Reply:
Great point! Absolutely

Reply:
Yup.  JSON Schema vocabularies were not really meant for constructing a whole new document format with embedded JSON Schemas.

Reply:
Thanksa henry, it was never meant to suggest the entire thing was a vocab. Made this clear now.